app/src/main/res/layout-land/topic_practice_subtopic.xml

```diff
@@ -31,17 +32,15 @@
       android:id="@+id/subtopic_check_box"
```

Comment:
It seems failing accessibility test.

Reply:
I am not getting what needs to be done?

Reply:
Clickable item should be 48dp or larger. In that case we are not supposed to give fixed height, so you can give min height as 48 dp and height as wrap_content. Make changes in both portrait and landscape to pass accessibility test.

Reply:
Okay